.github/workflows/terraform.yml

```diff
@@ -11,11 +11,11 @@ name: ♻ terraform
 on:
   workflow_call:
     inputs:
-      runs_on:
-        description: The label of the runner (GitHub- or self-hosted) to run this workflow on.
+      runner-image:
```

Comment:
Renaming an input would be a breaking change ([ref.](https://github.com/equinor/ops-actions/blob/f4f36e8d35440bb6e75d87e07faf9c77ea7d4b32/CONTRIBUTING.md#-reviewing-changes)), so a no-go in my opinion.

A better solution for keeping input names consistent could be to use the `runs_on` name for all workflows, instead of `runner-image`.

Reply:
ah noted, will make the appropriat changes 👍🏼